test/integration/model/create.test.js

```diff
@@ -964,7 +983,7 @@ describe(Support.getTestDialectTeaser('Model'), () => {
             return user.save({
               logging(sql) {
                 test = true;
-                expect(sql).to.contain('UPDATE "UserWithArrays" SET "myvals"=$1,"mystr"=$2,"updatedAt"=$3 WHERE "id" = $4');
+                expect(sql).to.contain('UPDATE "UserWithArrays" SET "myvals"=\'{}\',"mystr"=\'{}\'');
```

Comment:
Same thing as above, please assert on the complete SQL, avoid timestamps with `{ timestamps: false }` in the model.